content/en/docs/Endpoints/getTranscodeDecision.md

```diff
@@ -0,0 +1,141 @@
+---
+title: "getTranscodeDecision"
+linkTitle: "getTranscodeDecision"
+categories:
+- Transcoding
+description: >
+  Returns a transcode decision for a given media file.
+---
+
+`http://your-server/rest/getTranscodeDecision`
+
+Returns a transcode decision for a given media file. This endpoint is used by clients to determine if a media file should be transcoded before streaming. The server will analyze the provided client information and return a decision.
+
+### Parameters
+
+| Parameter | Req. | OpenS. | Default | Comment |
+| --- | --- | --- | --- | --- |
+| `songId` | **Yes** | | | The ID of the song to be transcoded. |
+| `offset` | No | | 0 | The time offset in seconds from which to start transcoding. |
+
+### Request Body
+
+The request body must be a JSON object containing the client's capabilities. See the [`ClientInfo`](../responses/clientinfo) documentation for more details.
+
+{{< tabpane persist=false >}}
+{{< tab header="**Example**:" disabled=true />}}
+{{< tab header="ClientInfo" lang="json">}}
+{
+  "name": "Pixel 8 Pro",
+  "platform": "Sonos",
+  "maxAudioBitrate": 512000,
+  "maxTranscodingAudioBitrate": 256000,
+  "directPlayProfiles": [
+  {
+"container": "mp3",
+"audioCodec": "mp3",
+"protocol": "http",
+"maxAudioChannels": 2
+  },
+  {
+"container": "flac",
+"audioCodec": "flac",
+"protocol": "*",
+"maxAudioChannels": 2
```

Comment:
In `DirectPlayProfile`, `protocol` is described as either `http` or `hls`. I understand the _intent_ of `*` here, but `*` is not specified as a valid value in the spec. If the intent is for it to be both `http` and `hls`, then I'd suggest either:
1. Make it an array
2. Codify `*` as allowing both methods